plugins/dependencies/src/main/kotlin/dependencies/Touchlab.kt

```diff
@@ -1,17 +1,17 @@
 @file:Suppress("PackageDirectoryMismatch", "SpellCheckingInspection", "unused")
 
 import de.fayard.refreshVersions.core.DependencyGroup
-import de.fayard.refreshVersions.core.DependencyNotationAndGroup
 import org.gradle.api.Incubating
 
-/**
- * Multithreaded Kotlin Multiplatform Utilities
- *
- * - GitHub page: [Stately](https://github.com/touchlab/Stately)
- * - [GitHub Releases here](https://github.com/touchlab/Stately/releases)
- */
 @Incubating
 object Touchlab {
+
+    /**
+     * Multithreaded Kotlin Multiplatform Utilities
+     *
+     * - GitHub repo: https://github.com/touchlab/Stately
+     * - Releases: https://github.com/touchlab/Stately/releases
```

Comment:
We are using markdown links so they are clickable right from the IDE that renders the KDoc.

```suggestion
     * [GitHub releases](https://github.com/touchlab/Stately/releases)
     *
     * GitHub repo: [touchlab/Stately](https://github.com/touchlab/Stately)
```